x/ibc/light-clients/07-tendermint/types/upgrade.go

```diff
@@ -154,3 +161,15 @@ func constructUpgradeConsStateMerklePath(upgradePath []string, lastHeight export
 	consPath = append(consPath, appendedKey)
 	return commitmenttypes.NewMerklePath(consPath...)
 }
+
+// ZeroedCustomFields returns true if all the client state fields have
+// been zeroed out.
+func ZeroedCustomFields(clientStateA *ClientState) bool {
```

Comment:
I would argue this should not be a helper function, given its used once and has too much name collision to `ZeroCustomFields`. Just put this logic above